spec/wash_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'wash'
+
+describe Wash do
+  describe 'pretty_print?' do
```

Comment:
Yeah, I was an idiot for making individual config fields helpers. It'd be nice to (at some point) refactor this into a hash that can be passed into a `Wash.config` type helper, but that's not that important right now.